### PR TITLE
Add Attendee data in Order Editor view

### PR DESCRIPTION
**Pull request by rafsuntaskin**
This is an Innovation Day work to improve the UX on the Admin Order screen.

**Before**
Screen Shot 2021-04-19 at 7 42 21 PM

**After**
Screen Shot 2021-04-19 at 7 38 44 PM

### CODE REVIEW

File: src/Main.php [hunk 1]
```diff
@@ -207,4 +207,136 @@ protected function get_attendee_meta( $ticket_id, $qr_ticket_id ) {
...
+	/**
+	 * Add inline styles for Attendee Table for Order Items.
+	 *
+	 * @since TBD
+	 */
+	function admin_order_table_styles() {
+
+		$custom_css = '
+                table.tribe-attendee-meta td:first-child {
+	                padding-left: 0px !important;
```
Comment (juanfra) [inline src/Main.php:333]: Do we need the `!important`? Is there a way we can add some more specificity?

Comment (juanfra) [reply src/Main.php:333]: When the value is `0px` I tend to prefer to use just `0`

**Review by juanfra — APPROVED** [review]
Looking good! 🎉

File: src/Main.php (hunk 1 above)
Comment (rafsuntaskin) [reply src/Main.php]: Updated

**Pull request merged**